jetty-server/src/main/java/org/eclipse/jetty/server/Request.java

```diff
@@ -2411,19 +2331,34 @@ else if (oldQueryParams == null || oldQueryParams.size() == 0)
         throw new ServletException("HttpServletRequest.upgrade() not supported in Jetty");
     }
 
-    public void setPathSpec(PathSpec pathSpec)
+    /**
```

Comment:
These were public methods on Request, should we be removing them in a point release? Maybe we should mark them as Deprecated and just not call them.

Reply:
We only used `setPathSpec(PathSpec pathSpec)` on a test case.
There's no practical way to use `Request.setPathSpec(PathSpec)` except in a testcase.